packages/big-design/src/components/Form/Description/Description.tsx

```diff
@@ -1,5 +1,36 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 
+import { LinkProps } from '../../Link';
 import { Small, TextProps } from '../../Typography';
 
-export const FormControlDescription: React.FC<TextProps> = ({ className, style, ...props }) => <Small {...props} />;
+import { StyledLink as FormControlDescriptionLink } from './styled';
```

Comment:
We should keep this consistent like our other components.
```suggestion
import { StyledLink } from './styled';
```